### ISSUE TITLE
The video in the readme doesn't seem to work

### ISSUE THREAD

**Issue by draeder** [hello-smile6/libv86-terminal-forwarding#3]
The video in the readme doesn't seem to work. Can you upload it to YouTube?

**Comment by hello-smile6** [comment]
> The video in the readme doesn't seem to work. Can you upload it to YouTube?

No, I cannot. If you're on Firefox, try installing a Chromium build. https://download-chromium.appspot.com/ Also, make sure you have MP4 codecs installed.

**Comment by hello-smile6** [comment]
@draeder I'd use my own video hosting, but it won't let me.

**Comment by hello-smile6** [comment]
> The video in the readme doesn't seem to work. Can you upload it to YouTube?

Does it work now?

**Comment by draeder** [comment]
It's a MacOS / Safari issue. It works in Chrome

**Comment by hello-smile6** [comment]
> It's a MacOS / Safari issue. It works in Chrome

Also affects some Linux distros using Firefox, @JaydenDev has issues with it
